06-func.md

```diff
@@ -65,47 +65,47 @@ and we have access to the value that we returned.
 
 > ## Integer division {.callout}
 >
-> We are using Python3 and integer division always return floating point.
+> We are using Python 3 and integer division always return floating point.
 >
 > ~~~ {.python}
-> $ python -c "print(5/9)"
+> $ python3 -c "print(5/9)"
 > ~~~
 > ~~~ {.output}
 > 0.5555555555555556
 > ~~~
 >
-> **Unfortunately**, in Python2 this isn't the case.
+> **Unfortunately**, in Python 2 this isn't the case.
 >
 > ~~~ {.bash}
-> $ python -c "print(5/9)"
+> $ python2 -c "print(5/9)"
 > ~~~
 > ~~~ {.output}
 > 0
 > ~~~
 >
-> If you are using Python2 and want a floating point as return of division
-> you need to use one of the following forms
+> If you are using Python 2 and want a floating point as return of division
+> you need to use one of the following forms:
 >
 > ~~~ {.bash}
-> $ python -c "print(float(5)/9)"
+> $ python2 -c "print(float(5)/9)"
 > ~~~
 > ~~~ {.output}
 > 0.555555555556
 > ~~~
 > ~~~ {.bash}
-> $ python -c "print(5/float(9))"
+> $ python2 -c "print(5/float(9))"
 > ~~~
 > ~~~ {.output}
 > 0.555555555556
 > ~~~
 > ~~~ {.bash}
-> $ python -c "print(5.0/9)"
+> $ python2 -c "print(5.0/9)"
 > ~~~
 > ~~~ {.output}
 > 0.555555555556
 > ~~~
 > ~~~ {.bash}
-> $ python -c "print(5/9.0)"
+> $ python2 -c "print(5/9.0)"
 > ~~~
 > ~~~ {.output}
 > 0.555555555556
```

Comment:
...and then a callout box: if you want an integer result from 4/2, you use `4//2`.